.github/workflows/deploy-asset-metadata.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       
       - name: Replace credentials
         run: |
-          sed -i "s/$GOOGLE_MAPS_API_KEY/${{ secrets.GOOGLE_MAPS_API_KEY }}/g" app.yaml
+          sed -i "s/$GOOGLE_MAPS_API_KEY/${{ secrets.GOOGLE_MAPS_API_KEY }}/g" asset-metadata/app.yaml
```

Comment:
Just realized that this can't work. You need to escape the first $, otherwise the shell will treat it as an environment variable and try to expand it.